### PR TITLE
draft crystal game

### PR DESCRIPTION
**Pull request by lsalazar2**

### CODE REVIEW

File: assets/javascript/game.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    wins=0;
+    losses=0;
+    var randomNum = getRandomInt(19,120); // Get a random number from 19-120 to compare total score with
+    console.log(randomNum);
+      
+    $("#score").text(randomNum); //store random number for current game's score
+
+    for (var i = 0; i < 4; i++) {
+      var crystals[i] = getRandomInt(1,9); //get 4 random #s for crystals 1-4
+
```
Comment (nkkwon) [inline assets/javascript/game.js:36]: Make sure you have the console open for chrome during development. I see that you have a syntax error somewhere around this line so always check console for errors.

**Review by nkkwon — CHANGES_REQUESTED** [review]

File: assets/javascript/game.js (hunk 1 above)
Comment (lsalazar2) [reply assets/javascript/game.js:36]: I updated this game over the break. Everything working. :)

**Comment by lsalazar2** [comment]
Updates dynamic crystals to work in flexbox as buttons.

**Comment by lsalazar2** [comment]
Updated dynamic crystal buttons to work.

**Pull request merged**